R/reportEmissions.R

```diff
@@ -30,125 +30,121 @@
+      
+      return(.x)
+    }
+    
+    # Emission from deforestation, degradation, and other land conversion
+    deforestation    <- collapseNames(dimSums(co2[, , "lu_deforestation"],    dim = "c_pools"))
+    degradation      <- collapseNames(dimSums(co2[, , "lu_degrad"],           dim = "c_pools"))
+    other_conversion <- collapseNames(dimSums(co2[, , "lu_other_conversion"], dim = "c_pools"))
+    regrowth         <- collapseNames(dimSums(co2[, , "lu_regrowth"],         dim = "c_pools"))
+    
+    # TODO How does the LTS change the pools here? 
+    # Above Ground / Below Ground Carbon
+    totalPools   <- collapseNames(dimSums(co2[, , "total"], dim = "land"))
+    climatePools <- collapseNames(dimSums(co2[, , "cc"],    dim = "land"))
+    landusePools <- collapseNames(dimSums(co2[, , "lu"],    dim = "land"))
+    
+    # Calculate wood products, if forestry module was activated and desired
+    emisWoodProducts <- carbonLTS(gdx, level = "regglo", unit = "gas", cumulative = .cumulative)[, getYears(totalNetFlux), ]
+    
+    if (!is.null(emisWoodProducts) && storageWood) {
+      
+      if (.cumulative) {
+        # Can't divide by 1000 during cumulative calc as in def runs its NULL and NULL/1000 is numeric(0)
+        emisWoodProducts <- emisWoodProducts / 1000
       }
+      
+      # All categories
+      inflow  <- collapseNames(emisWoodProducts[, , "annual_inflow"])
+      outflow <- collapseNames(emisWoodProducts[, , "annual_outflow"])
+      
+      # Wood products (not including constr wood)
+      emisWood        <- collapseNames(emisWoodProducts[, , "emis_wood"])        # -1 removed in carbonLTS_IPCC.R
+      emisWoodFuel    <- collapseNames(emisWoodProducts[, , "emis_woodfuel"])    # -1 removed in carbonLTS_IPCC.R
+      emisConstrWood  <- collapseNames(emisWoodProducts[, , "emis_constr_wood"]) # -1 removed in carbonLTS_IPCC.R
+      
+      # Purely industrial roundwood
+      emisWoodInflow  <- collapseNames(emisWoodProducts[, , "wood_inflow"]) # inflow is negative emissions
+      emisWoodOutflow <- collapseNames(emisWoodProducts[, , "wood_outflow"])
+      emisWoodNet     <- collapseNames(emisWoodInflow + emisWoodOutflow) 
+      
+      # Building materials
+      emisBuildingInflow  <- collapseNames(emisWoodProducts[, , "building_inflow"]) # inflow is negative emissions
+      emisBuildingOutflow <- collapseNames(emisWoodProducts[, , "building_outflow"])
+      emisBuildingNet     <- collapseNames(emisBuildingInflow + emisBuildingOutflow) 
+      
+      # Sum of net emissions from industrial roundwood and building material
+      storage <- emisWoodNet + emisBuildingNet
+      
+      # Total emissions from wood harvest
+      wood <- emisWoodFuel + storage # emisWood is already accounted for in storage
+      
+      # remove negative emissions from long-term storage of wood products
+      harvestOld <- dimSums(co2[, , "lu_harvest"], dim = 3)
+      harvestNew <- harvestOld - (emisWoodInflow + emisBuildingInflow)
```

Comment:
Florian, could you check this logic? If my interpretation is correct, then this is all ready to push.

Reply:
I think `emisWoodInflow` and `emisBuildingInflow` are negative (previously I thought they are positively defined)
Therefore it should be + and not - 
but better check by running the carbonLTS function

Reply:
Good. That resolves my confusion, thank you.